src/Parser/YaccParser.mly

```diff
@@ -4,13 +4,16 @@
 
 /** Yacc-generated parser */
 
-/* Author: Piotr Polesiuk, 2023,2024 */
+(* 2023,2024: Piotr Polesiuk: initial implementation
+   2024: Jakub  Chomiczewski: operators*)
 
 %token<string> LID UID TLID
+%token<string> OP_0 OP_20 OP_30 OP_40 OP_50 OP_60 OP_70 OP_80 OP_90 OP_100
+%token<string> OP_225 
 %token<int> NUM
 %token<string> STR
 %token BR_OPN BR_CLS SBR_OPN SBR_CLS CBR_OPN CBR_CLS
-%token ARROW ARROW2 BAR COLON COMMA DOT EQ SEMICOLON2 SLASH
+%token ARROW ARROW2 BAR COLON COMMA DOT EQ SEMICOLON2 SLASH SEMICOLON
```

Comment:
Why the `SEMICOLON` is a separate token? Isn't it just a regular operator?

Reply:
I added this token because I am using this in op_0.

Reply:
But it is completely opposite: it is used in `op_0` because it is a separate token. It can be removed from the list of tokens and it would simplify lexer and `op_0` grammar.